src/resources/js/events-admin.js

```diff
@@ -778,6 +778,14 @@ jQuery( function( $ ) {
 			.parent()
 			.addClass( 'current' );
 	}
+	if ( $( 'body' ).hasClass( 'post-type-tec_calendar_embed') ) {
```

Comment:
Make known to WP the parent file like [this](https://github.com/the-events-calendar/event-tickets/blob/15f56a9a3d97f5cdd8f89835d692b2533981d887/src/Tickets/Commerce/Admin/Singular_Order_Page.php#L58).

Dont try to hack your way into this through JS - can cause other issues.